linode/manager
# refactor: [M3-5457] - Make HelpIcon more generic

## Description 📝

Rename the `<HelpIcon />` component to `<TooltipIcon />`.

Replace the former `isError` prop which was a boolean value, with the new `status` prop which allows a constant value to be passed in and render the appropriate SVG icon. The values accepted are `SUCCESS`, `ERROR`, `WARNING`, `INFO`, and `HELP`.

Also, replace CSS classes with the introduction of the `sxTooltipIcon` prop which takes advantage of the new MUI v5 `sx` prop to define specific styling.

## Preview 📷

https://user-images.githubusercontent.com/119514965/228618002-3827c525-9e14-40a3-9fee-23c9986f1cb9.mov




## How to test 🧪
On your local instance of Cloud Manager, visit the the billing landing page and hit the 'Add Payment Method' button.
Next, open the Browser DevTools and go to the 'Components' tab. Select either the GoogleChip or PayPal chip component and toggle it's `state` from `false` to `true` and inspect that an red error icon is rendered.

Additionally, inspect that all former help icons are rendered as expected.
